### PR TITLE
chore(modal): remove storybook multiple exports

### PR DESCRIPTION
**Pull request by anuraghazra**
Closes # 

## 📝 Description

Removed multiple reexports of modal.stories & fixed the import order of `react-lorem-component`

> Add a brief description

## ⛳️ Current behavior (updates)

> Please describe the current behavior that you are modifying

## 🚀 New behavior

> Please describe the behavior or changes this PR adds

### CODE REVIEW

File: packages/modal/stories/modal.stories.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import { useDisclosure } from "@chakra-ui/hooks"
 import { chakra } from "@chakra-ui/system"
 import * as React from "react"
+//@ts-ignore
+import Lorem from "react-lorem-component"
+
```
Comment (anuraghazra) [inline packages/modal/stories/modal.stories.tsx:6]: fixes eslint error

```
`react-lorem-component` import should occur before import of `../src`eslintimport/order
```

File: packages/modal/stories/modal.stories.tsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import { useDisclosure } from "@chakra-ui/hooks"
 import { chakra } from "@chakra-ui/system"
 import * as React from "react"
+//@ts-ignore
```
Comment (with-heart) [inline packages/modal/stories/modal.stories.tsx:4]: I know you didn't add this, but would you mind removing it? Shouldn't be necessary now.

**Review by with-heart — APPROVED** [review]

File: packages/modal/stories/modal.stories.tsx (hunk 2 above)
Comment (anuraghazra) [reply packages/modal/stories/modal.stories.tsx:4]: Oh okay yeah I'll remove it.

Comment (with-heart) [reply packages/modal/stories/modal.stories.tsx]: Thank you! Will get this merged today

**Pull request merged**